devices/AtModem/Http/Http/HttpHeaders.cs

```diff
@@ -0,0 +1,81 @@
+﻿//
+// Copyright (c) .NET Foundation and Contributors
+// Portions Copyright (c) Microsoft Corporation.  All rights reserved.
+// See LICENSE file in the project root for full license information.
+//
+
+using System.Diagnostics;
+
+namespace System.Net.Http.Headers
+{
+    /// <summary>
+    /// Key/value pairs of headers. The value is either a raw <see cref="string"/> or a <see cref="HttpHeaders.HeaderStoreItemInfo"/>.
```

Comment:
We don't have `HeaderStoreItemInfo`

Reply:
I've cleaning as much as I could find. Those files are copy/paste for convenience. But agree, comments should be as close as the reality.